### PR TITLE
Add a getting started section to README.md

### PR DESCRIPTION
**Pull request by MartinDelille**

### CODE REVIEW

**Comment by konserw** [comment]
nice! :+1:

File: README.md [hunk 1]
```diff
@@ -63,3 +63,23 @@ start build\examples\Calc\BoostCalculatorSteps.exe
...
+```
+host: localhost
+port: 3902
+```
+
+Create your first feature (an example is available [here](examples/Calc/features/addition.feature)).
+
+Then create your step definition runner (an example is available [here](examples/Calc/features/step_definitions/BoostCalculatorSteps.cpp)). In order to compile the step definition runner, make sure to add [cucumber include directory](includes) to the include path and link with *libcucumber-cpp.a* and additional testing libraries (boost unit test).
+
+Run the step definition runner in the background (adding an ampersand), then `cucumber`. Your feature should be executed (and the step definition runner exit).
```
Comment (paoloambrosio) [inline README.md]: I would rephrase it in a more generic way (not Linux/OSX specific) like...

Run the step definition runner in the background and then cucumber, like in the Calc example in the previous section. The step definition runner should exit after the feature is run and cucumber exits.

**Comment by paoloambrosio** [comment]
Very nice indeed!
